test_scripts/API/VehicleData/WindowStatus/017_Resumption_of_subscription_for_2_apps_to_the_windowStatus_param_after_an_Unexpected_Disconnect.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0261-New-vehicle-data-WindowStatus.md
+-- Description: Check that SDL resumes the subscription for 'windowStatus' parameter for two Apps after unexpected disconnect/connect.
+-- Precondition:
+-- 1) Two apps are registered and activated.
+-- 2) Apps are subscribed to `windowStatus` data.
+-- 3) Unexpected disconnect and reconnect are performed.
+-- In case:
+-- 1) Mobile app1 and app2 register with actual hashID
+-- SDL does:
+--  a) start data resumption for both apps
+--  b) resume the subscription and sends VI.SubscribeVD request to HMI
```

Comment:
It needs to be explicitly defined that `VI.SubscribeVD ` shouldn't be sent for 2nd app